pages/common/acme.sh-dns.md

```diff
@@ -0,0 +1,25 @@
+# acme.sh --dns
+
+> Use a DNS-01 challenge to issue a TLS certificate.
+> DNS API mode documentation: <https://github.com/acmesh-official/acme.sh/wiki/dnsapi>.
+> More information: <https://github.com/acmesh-official/acme.sh/wiki/DNS-manual-mode>.
```

Comment:
I'm not sure if we should use two seperate links for that. What do you think about:
```suggestion
> More information: <https://github.com/acmesh-official/acme.sh/wiki>.
```

Reply:
I just saw that `acme.sh.md` uses `https://github.com/acmesh-official/acme.sh`. I'm not sure if we should prefer the wiki or the Readme, but we should at least use the same link for both pages.

Reply:
But wiki details the DNS mode much better (containing no less than four detailed pages), while the README details the root acme.sh command much better. I am really happy with that as it is.

Reply:
In that case it's fine of course